ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/exceptionhandling/AdminRestResponseEntityExceptionHandler.java

```diff
@@ -48,7 +49,8 @@ private ResponseEntity<Object> handleException(
 			RuntimeException ex, HttpStatus status, WebRequest request) {
 		log.error(ex.getMessage());
 		String bodyOfResponse = ex.getMessage();
-		return handleExceptionInternal(ex, bodyOfResponse,
-				new HttpHeaders(), status, request);
+		var httpHeaders = new HttpHeaders();
+		httpHeaders.setContentType(MediaType.TEXT_PLAIN);
```

Comment:
It would be nice if this was "custom" as well. For example, the `ShaclValidationException` returns the shacl validation errors in turtle, in that case we would want turtle here instead of text_plain. If we have no information, text_plain is a good default. Maybe you can include the content type in the exception and fall back to plain text when no content type is available because for parsing issues with jena, we will have plain_text and there will always be exceptions that we did not foresee.